index.html

```diff
@@ -0,0 +1,448 @@
+            <p class="txt">Hello I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <div class="supporting-text">
+                <a class="connect" href="#">LET’S CONNECT</a>
+            </div>
+
+            <ul class="frame1">
+                <li><a href="#"><img class="github2" src="images/github.png"></a></li>
+                <li><a href="#"><img class="linkedin2" src="images/Linkedin icon.png"></a></li>
+                <li><a href="#"><img class="vector2" src="images/Vector.png"></a></li>
+                <li><a href="#"><img class="twitter2" src="images/twitter.png"></a></li>
+                <li><a href="#"><img class="m2" src="images/m.png"></a></li>
+            </ul>
+
+            <button type="button" class="resume">
+                <span>Get my resume</span>
+            </button>
+
+        </div>
+
+        <div class="skill-list">
+            <div class="frame12">
+                <ul class="frame7">
+                    <li class="lang">Languages</li>
+                    <li class="arrow-down"><i class="fa-solid fa-chevron-down"></i></li>
+                </ul>
+            </div>
+
+            <div class="language-list">
+                <ul class="frame9">
+                    <li class="js-ellipse">
+                        <a href="#"><img src="images/js.png"></a>
+                    </li>
+                    <li class="script">JavaSript</li>
+                </ul>
+                <ul class="frame10">
+                    <li class="html-ellipse">
+                        <a href="#"><img src="images/html.png"></a>
+                    </li>
+                    <li class="script2">HTML</li>
+                </ul>
+                <ul class="frame11">
+                    <li class="css-ellipse">
+                        <a href="#"><img src="images/css.png"></a>
+                    </li>
+                    <li class="script3">CSS</li>
+                </ul>
+            </div>
+        </div>
+        <ul class="frame-8">
+            <li class="script4">Frameworks</li>
+            <li class="arrow-right"><i class="fa-solid fa-chevron-right"></i></li>
+        </ul>
+     <hr class="hr">
+        <ul class="frame-9">
+            <li class="script5">Skills</li>
+            <li class="arrow-right2"><i class="fa-solid fa-chevron-right"></i></li>
+        </ul>
+        <hr class="hr">
+
+    </section>
+    <section class="contact-form">
+             
+                <div class="contain" >
+        <h2 id="contact-me">Contact me</h2>
+        <p class="contact-p">If you have an application you are interested in developing, a feature that you need built
+            or a project that needs coding. I’d love to help with it </p>
+
+        <form action="https://formspree.io/f/mdojqvjo" method="POST">
```

Comment:
- [ Required ] It would be better if you removed the horizontal scroll on your mobile design, on mobile devices, the user should not be able to scroll horizontally. I had the experience captured below when I tried to scroll horizontally.
I noticed that this might be caused the `widths`, `padding` and `margin` of some elements, please consider having this CSS rule at the top of your stylesheet. Please read this article to understand what it does [https://www.w3schools.com/css/css3_box-sizing.asp](https://www.w3schools.com/css/css3_box-sizing.asp)

```CSS
  *{
      box-sizing: border-box;
   }
```

https://user-images.githubusercontent.com/16392046/221828282-7d5311dd-ffd5-43ae-8598-9f094999d54b.mp4

Reply:
Thanks @Felix45 , I will work on that all